helm-chart/metabase/templates/deployment.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       serviceAccountName: {{ include "metabase.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.initContainers }}
+      initContainers:
```

Comment:
what is this line for?

Reply:
If there's something useful for the community feel free to include it, but add the relevant documentation about what's this for and how to use it. Also please include documentation about how to pass values to the extraVolumes map